src/main/java/org/broadinstitute/hellbender/tools/walkers/CombineGVCFs.java

```diff
@@ -81,7 +82,7 @@
      * Which groups of annotations to add to the output VCF file.
      */
     @ArgumentCollection
-    public VariantAnnotationArgumentCollection variantAnnotationArgumentCollection = new VariantAnnotationArgumentCollection(
+    public DefaultGATKVariantAnnotationArgumentCollection variantAnnotationArgumentCollection = new DefaultGATKVariantAnnotationArgumentCollection(
```

Comment:
I think the type of variantAnnotationArgumentCollection in all of these tools should be GATKVariantAnnotationArgumentCollection, not DefaultGATKVariantAnnotationArgumentCollection. Its only the actual object that is created that should be DefaultGATKVariantAnnotationArgumentCollection.

It's somewhat moot, since once the descriptor is fully integrated, these argument collections that are embedded in the tools will go away altogether, and will be replaced by the DefaultGATKVariantAnnotationArgumentCollection that is passed to the descriptor when its created. From then on, it will be used through the GATKVariantAnnotationArgumentCollection interface. So in the meantime it seems more correct to have the type of these be the interface, not the implementation, since that matches the direction in which we're headed. Feel free to push back if I'm missing something.

Reply:
I agree with you, I *THINK* I caught all of the DefaultGATKVariantAnnotationArgumentCollection uses but I might have missed one somewhere. You are right, in the next branch of this work I have gotten rid of all of the uses of these argument collections in the code entirely so this is somewhat a moot point but I see value.